backend/routes/import_routes.py

```
@@ -0,0 +1,109 @@
+            created=stats["created"],
+            updated=stats["updated"],
+            skipped=stats["skipped"],
+            errors=0,
+        )
+
+    except Exception as exc:
+        logger.error("SAP CSV import failed: %s", exc, exc_info=True)
+        raise HTTPException(
+            status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+            detail=f"Import failed: {exc}",
+        )
```

Ruff (0.15.9)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)